vic/drivers/classic/src/read_soilparam.c

```diff
@@ -692,6 +696,108 @@ read_soilparam(FILE *soilparam,
             temp.Ws = temp.Ws / temp.max_moist[layer];
         }
 
+        // Soil thermal node thicknesses and positions
```

Comment:
It seems that currently the last section (around the last ~15 lines) in `initialize_soil.c` also derives some `soil_con` values. Should that part be moved here to `read_soilparam()` also?